packages/liquidator/test/Liquidator.js

```diff
@@ -1272,185 +1272,10 @@ contract("Liquidator.js", function(accounts) {
-              const withdrawLiveness = empProps.withdrawLiveness.toNumber();
-              let nextTime = Math.ceil(
-                Number(sponsor1Positions.withdrawalRequestPassTimestamp) - withdrawLiveness * 0.5
-              );
-              await emp.setCurrentTime(nextTime);
-
-              priceFeedMock.setCurrentPrice(convertPrice("1"));
-              await liquidator.update();
-              await liquidator.liquidatePositions();
-              assert.equal(spy.callCount, 1); // There should be one log from the liquidation event of the withdrawal.
-
-              // There should be exactly one liquidation in sponsor1's account. The liquidated collateral should be the original
-              // amount of collateral minus the collateral withdrawn. 125 - 10 = 115
-              let liquidationObject = (await emp.getLiquidations(sponsor1))[0];
-              assert.equal(liquidationObject.sponsor, sponsor1);
-              assert.equal(liquidationObject.liquidator, liquidatorBot);
-              assert.equal(liquidationObject.state, LiquidationStatesEnum.PRE_DISPUTE);
-              assert.equal(liquidationObject.liquidatedCollateral, convertCollateral("115"));
-              assert.equal(liquidationObject.lockedCollateral, convertCollateral("125"));
-
-              // Advance the timer to the liquidation expiry.
-              const liquidationTime = liquidationObject.liquidationTime;
-              const liquidationLiveness = 1000;
-              await emp.setCurrentTime(Number(liquidationTime) + liquidationLiveness);
-
-              // Now that the liquidation has expired, the liquidator can withdraw rewards.
-              const collateralPreWithdraw = await collateralToken.balanceOf(liquidatorBot);
-              await liquidator.update();
-              await liquidator.withdrawRewards();
-              assert.equal(spy.callCount, 2); // 1 new info level events should be sent at the conclusion of the withdrawal. total 2.
-
-              // Liquidator should have their collateral increased by Sponsor1's collateral.
-              const collateralPostWithdraw = await collateralToken.balanceOf(liquidatorBot);
-              assert.equal(
-                toBN(collateralPreWithdraw)
-                  .add(toBN(convertCollateral("125")))
-                  .toString(),
-                collateralPostWithdraw.toString()
-              );
-
-              // Liquidation data should have been deleted.
-              assert.deepStrictEqual(
-                (await emp.getLiquidations(sponsor1))[0].state,
-                LiquidationStatesEnum.UNINITIALIZED
-              );
-
-              // The other two positions should not have any liquidations associated with them.
-              assert.deepStrictEqual(await emp.getLiquidations(sponsor2), []);
-            }
-          );
-          versionedIt([{ contractType: "any", contractVersion: "any" }])(
-            "should trigger and only submit the emp minimum position",
```

Comment:
@daywiss This test currently checks that the liquidator only has `balance - whaleDefenseFundWei` tokens to possibly liquidate with and therefore can only submit min liquidations on large enough positions. So this test isn't relevant